app/src/main/java/de/grobox/transportr/networks/TransportNetworks.java

```diff
@@ -635,6 +635,25 @@
 					.setGoodLineNames(true)
 					.build(),
 
+			// Nicaragua
+
+			new TransportNetworkBuilder()
+					.setId(NetworkId.MANAGUA)
+					.setName(R.string.np_name_managua)
```

Comment:
Where do you defined this strings?

Reply:
In the strings.xml (now added after a comment by @ialokim)